internal/pkg/block/block_test.go

```diff
@@ -62,7 +62,7 @@ func TestTriangleEncoding(t *testing.T) {
 		// pass when non-zero values do not due to nil/null encoding.
 		candidate1 := blk.NewEPoStCandidate(5, []byte{0x05}, 52)
 		candidate2 := blk.NewEPoStCandidate(3, []byte{0x04}, 3000)
-		postInfo := blk.NewEPoStInfo([]byte{0x07}, []byte{0x02, 0x06}, candidate1, candidate2)
+		postInfo := blk.NewEPoStInfo([]blk.EPoStProof{blk.NewEPoStProof(abi.RegisteredProof_StackedDRG2KiBPoSt, []byte{0x07})}, []byte{0x02, 0x06}, candidate1, candidate2)
```

Comment:
Please add a constant `DevPoStProof` aligned with `DevSectorSize` and use that rather than repeat the specific value `abi.RegisteredProof_StackedDRG2KiBPoSt ` here a lot.

Reply:
done